token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -398,6 +398,23 @@ There are **Primitive Rules** and **Composed Rules** that are created by combini
 
 More details of the Token Authorization Rules program, including examples, can be found [here](https://github.com/metaplex-foundation/mpl-token-auth-rules/blob/main/README.md).
 
+### Token Metadata Operations subject to Authorization Rules
+
+Several operations involving `pNFT` on Token Metadata are subject to Token Authorization Rules – depending of the rule configured, the operation will be authorized or not. The creator (`update authority`) of an asset has the flexibility to manage these rules through the [`ProgrammableConfig`](https://github.com/metaplex-foundation/metaplex-program-library/blob/ad5f39c465676299951c91f8cf9216812b884531/token-metadata/program/src/state/metadata.rs#L364-L380) on a Metadata account.
+
+The list of operations are:
+
+- `Transfer:WalletToWallet`: operation representing a transfer between wallets (currently not in use)
+- `Transfer:Owner`: operation representing a transfer initiated by the owner of the asset
+- `Transfer:MigrationDelegate`: operation representing a transfer initiated by a `Migration` delegate
+- `Transfer:SaleDelegate`: operation representing a transfer initiated by a `Sale` delegate
+- `Transfer:TransferDelegate`: operation representing a transfer initiated by a `Transfer` delegate
+- `Delegate:LockedTransfer`: operation representing the request to approve a `LockedTransfer` delegate, a delegate that can locked and transfer an asset to a predefined address
+- `Delegate:Transfer`: operation representing the request to approve a `Transfer` delegate, a delegate that can transfer an asset
+- `Delegate:Utility`: operation representing the request to approve a `Utility` delegate, a delegate that can lock and burn an asset
+- `Delegate:Staking`: operation representing the request to approve a `Staking` delegate, a delegate that can lock an asset
+- `Delegate:Sale`: operation representing the request to approve a `Sale` delegate, a delegate that can transfer an asset and disable transfers by the owner while the delegate is in place
```

Comment:
I think this makes sense to list the ones we actually have and the one we intend to use.  I'm sure you know this, but the current Metaplex default has some extra ops that may never be used (like `Delegate:Use`) or are fully outdated (like `Delegate:Update`).  We probably don't need to mention any of that stuff right?  Best to keep this section focused on what TM actually has or intends to have in the future?

### Current default
```
[
  1,
  "ELskdHjzTQ6F4bBibhk4iqy63gSPs8ELec9HbfAaSDJk",
  "Metaplex Foundation Rule Set",
  {
    "Transfer:WalletToWallet": "Pass",
    "Transfer:Owner": "Pass",
    "Transfer:MigrationDelegate": "Pass",
    "Transfer:SaleDelegate": "Pass",
    "Transfer:TransferDelegate": "Pass",
    "Delegate:LockedTransfer": "Pass",
    "Delegate:Update": "Pass",
    "Delegate:Transfer": "Pass",
    "Delegate:Utility": "Pass",
    "Delegate:Staking": "Pass",
    "Delegate:Authority": "Pass",
    "Delegate:Collection": "Pass",
    "Delegate:Use": "Pass",
    "Delegate:Sale": "Pass"
  }
]
```